server/engine/src/passes/dynamics_pass.py

```diff
@@ -0,0 +1,271 @@
+import nimblephysics as nimble
```

Comment:
Would it make sense to rename this folder `dynamics_pass`, since these three passes essentially serve the dynamics fitting process?